src/cli/constants.js

```diff
@@ -104,7 +104,7 @@ ${sec('Helper commands')}
 `
 };
 
-const NEW_SHORTHAND_EXPANSION = ['generate', 'project'];
+const NEW_SHORTHAND_EXPANSION = ['generate', 'project', '--'];
 const INIT_SHORTHAND_EXPANSION = NEW_SHORTHAND_EXPANSION.concat(['--', '--here']);
```

Comment:
Does `aunty init` still work, or do we need to remove the 2nd `--` here now?

Reply:
`aunty init` appears to still work as it used to.